packages/bundle/package.json

```
@@ -29,53 +29,53 @@
     "start:webpack": "npm run build:webpack -- --watch"
   },
   "dependencies": {
-    "@babel/runtime": "7.12.5",
+    "@babel/runtime": "^7.13.10",
```

For all production dependencies, we pinned them down (`--exact` or `--save-exact`)

`lerna add --exact @babel/runtime` or `npm install --save-exact @babel/runtime`.

```suggestion
    "@babel/runtime": "7.13.10",
```